consensus/dummy/consensus.go

```diff
@@ -28,10 +28,9 @@ import (
 var (
 	allowedFutureBlockTime = 10 * time.Second // Max time from current time allowed for blocks, before they're considered future blocks
 
-	ErrInsufficientBlockGas = errors.New("insufficient gas to cover the block cost")
-
-	errInvalidBlockTime  = errors.New("timestamp less than parent's")
-	errUnclesUnsupported = errors.New("uncles unsupported")
+	errInvalidBlockTime    = errors.New("timestamp less than parent's")
+	errUnclesUnsupported   = errors.New("uncles unsupported")
+	errInvalidBlockGasCost = errors.New("invalid blockGasCost")
```

Comment:
I think my final nit: Most of the changes to this file don't seem necessary or directly related to this PR. Can we limit it to only match the changes made to `consensus.go` in the [coreth PR](https://github.com/ava-labs/coreth/pull/1168/files#diff-532a2c6a5365d863807de5b435d8d6475552904679fd611b1b4b10d3bf4f5010) and just pass `nil` for `contribution`?

I think that would make the diff of this PR as minimal as possible